pkg/client/cli/cliutil/flagtypes.go

```diff
@@ -121,9 +139,9 @@ var flagTypes = map[TypeEnum]interface{}{
 	"int32": func(fs *pfs, x *int32) Value { fs.Int32Var(x, "x", *x, ""); return simple(fs) },
 	"int64": func(fs *pfs, x *int64) Value { fs.Int64Var(x, "x", *x, ""); return simple(fs) },
 
-	"int-slice":   func(fs *pfs, x *[]int) Value { fs.IntSliceVar(x, "x", *x, ""); return slice(fs, x) },
-	"int32-slice": func(fs *pfs, x *[]int32) Value { fs.Int32SliceVar(x, "x", *x, ""); return slice(fs, x) },
-	"int64-slice": func(fs *pfs, x *[]int64) Value { fs.Int64SliceVar(x, "x", *x, ""); return slice(fs, x) },
+	"intSlice":   func(fs *pfs, x *[]int) Value { fs.IntSliceVar(x, "x", *x, ""); return slice(fs, x) },
```

Comment:
these camelCasing changes were necessary to line up with the type reported by pflag